sirbot/pythondev/slack.py

```diff
@@ -470,3 +472,13 @@ def add(self, slack):
                             " `{0}`".format(command.text)
 
         await slack.send(response)
+
+        async def members_joined(self, event, slack, _):
+            if event['channel'] == await slack.channels.get(name='general').id:
+                await slack.channels.get('general', update=True)
+                members = await slack.channels.get(name='general').members
+                if (len(members) % 1000) == 0:
+                    to = await slack.channels.get(name='general').members
+                    message = SlackMessage(to=to)
+                    message.text = 'We have just reached {} members!'.format()
```

Comment:
Ah yes... whoops!  I was hoping I could print our what sort of thing the slack.channels.get returned, but I couldn't test it - I think .members returns a list of members.  I didn't know I could just sent to the channel directly.  Ok, thanks for this - I'll make a few changes and update it, thanks @ovv!